docs/glossary.md

```diff
@@ -0,0 +1,79 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common Aliases.
+
+## CustomResourceDefinitions (CRDs)
+
+The following [CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) are defined by the OLM.
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion represents a particular version of a ClusterService and its operator. It includes metadata such as name, description, version, repository link, labels, icon, etc. It declares `owned`/`required` CRDs, cluster requirements, and install strategy that tells OLM how to create required resources and set up the operator as a [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).
+**Aliases**: CSV(s)
```

Comment:
style nit: I think we want an extra newline above `**Aliases**` so it doesn't get appended to the previous paragraph